src/main/scala/org/hammerlab/guacamole/Common.scala

```diff
@@ -156,15 +158,18 @@ object Common extends Logging {
   def loadReadsFromArguments(
     args: Arguments.Reads,
     sc: SparkContext,
-    filters: Read.InputFilters): ReadSet = {
-
-    ReadSet(
-      sc,
-      args.reads,
-      filters,
-      contigLengthsFromDictionary = !args.noSequenceDictionary,
-      config = ReadLoadingConfigArgs.fromArguments(args)
-    )
+    filters: Read.InputFilters): (RDD[MappedRead], ContigLengths) = {
```

Comment:
Good point; all callers that load just one set of reads currently filter to mapped reads as well, so I was trying to expose a convenient interface to them.

I've changed this so that there are now special-case helpers in `ReadSets` for single-read-set cases ("mapped" and "all"), and removed `ReadSet` altogether.